apps/studio/src/components/tableinfo/TableSchema.vue

```diff
@@ -417,6 +420,18 @@ export default Vue.extend({
         data: this.tableData,
         placeholder: "No Columns",
       })
+    },
+    columnNameCellClick(_e: any, cell: CellComponent) {
```

Comment:
If you click away from the table does it unselect the cell? Take a look at table table for how to do that.